locales/en.default.schema.json

```diff
@@ -1709,6 +1709,87 @@
         "name": "Email signup"
       }
     },
+    "email-signup-banner": {
+      "name": "Email signup banner",
+      "settings": {
+        "paragraph": {
+          "content": "Each email subscription creates a customer account. [Learn more](https://help.shopify.com/en/manual/customers)"
+        },
+        "image": {
+          "label": "Background image"
+        },
+        "show_background_image": {
+          "label": "Show background image"
+        },
+        "desktop_text_box_position": {
+          "options__1": {
+            "label": "Top"
+          },
+          "options__2": {
+            "label": "Center"
+          },
+          "options__3": {
+            "label": "Bottom"
+          },
+          "label": "Desktop text position"
+        },
+        "show_text_box": {
+          "label": "Show text box on desktop"
+        },
+        "image_overlay_opacity": {
+          "label": "Image overlay opacity"
+        },
+        "color_scheme": {
```

Comment:
I would expect the color scheme to still influence the font color even when the text box background is removed.

Here for example it should be black:
![](https://screenshot.click/21-10-n4xp2-6v587.png)

Reply:
Because of the opacity overlay, the text has to remain white.

Reply:
Ah i see. And on the other hand, if you are using a light coloured image you will have to use the overlay opacity or text box background in order to see the text properly. Got it 👍

Reply:
Out of scope for this PR, but I agree with Ludo here. We should not hardcode color values here. I think the color system could work perfectly fine with an overlay opacity slider as well.